akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala

```diff
@@ -37,14 +37,29 @@ object BackoffSupervisor {
    * @param randomFactor after calculation of the exponential back-off an additional
    *   random delay based on this factor is added, e.g. `0.2` adds up to `20%` delay.
    *   In order to skip this additional delay pass in `0`.
+   * @param replyWhileStopped The message that the supervisor will send in response to all messages while
+   *   its child is stopped.
+   */
+  def props(
```

Comment:
isn't it enough to use the `def props(options: BackoffOptions): Props`
I think this method was the first implementation but then we added BackoffOptions, which are easier to use and maintain